### ISSUE TITLE
Windows cannot be nested (sub-windows, dialogs)

### ISSUE THREAD

**Issue by LoganDark** [slint-ui/slint#611]
Having a Window as a child of another Window just causes its children to show up on top of the first Window. Windows should be allowed to have other Windows as children and those other windows should actually show up as separate windows.

**Comment by ogoffart** [comment]
Duplicate of https://github.com/slint-ui/slint/issues/6607
(Still pending on winit 0.31)
